chatbot.html

```diff
@@ -358,39 +358,319 @@ <h2>DIETBOT</h2>
+            <p>Food banks provide avenues to donate your food waste that you do not need anymore. With this simple act, you can also help and benefit those who need it more with sufficient amount food to feed for themselves and their family <br>
+            You take your part in helping out the needy and contribute to donating food to them!</p>
+          </div>
+        </div>
+        <div id="leftover" class="tabcontent" style="background:url('website_images/cutting_vegetables.webp') no-repeat center center fixed">
+          <div class="tab-desc">
+            <h2>4. Cook up some delicious leftover recipes</h2>
+            <p>Got any leftovers to spare? Well don't throw them away! <br> You can put these dishes to good use. Cook up a recipe using ingredients from your leftovers. </p>
+            <p>Even if the ingredients may not seem to be in its shape or appearance, <br> there is still a chance that those ingredients can still be used!</p>
+          </div>
+        </div>
+        <div id="store" class="tabcontent" style="background:url('website_images/food-in-fridge.webp') no-repeat center center fixed; overflow: scroll; overflow-x: hidden; background-size: 100%">
+          <div class="tab-desc">
+            <h2>5. Proper storage of food</h2>
+            <p><b>CHECK!</b> Did you store your food correctly? Here are some checks you should know and conduct for certain types of food products to improve their shelf-life:  </p>
+            <p>Here you can see a picture below of where food are stored: </p>
+          </div>
+          <img src="website_images/fridge.jpg" alt="fridge" class="center">
+          <div class="tab-desc">
+            <p>Did you know that certain types of fruits and vegetables needs to have a certain and suitable storage environment to sustain its shelf life?</p>
+            <p>Here are examples of where you should store certain kinds of fruits and vegetables: </p>
+          </div>
+            <img src="website_images/f&v-tables.png" alt="table" class="center">
+        </div>
+
+        <div id="avoid" class="tabcontent" style="background:url('website_images/food-waste.jpeg') no-repeat center center fixed; background-size: 100%;">
+          <div class="tab-desc">
+            <h2>6. Avoid going for too much food</h2>
+            <p>Lastly, planning of what amount of food is enough and sufficient for you in the long run.</p>
+            <p>Before spending on buying more food, think first if it is neccessary to buy first and whether it is enough to suit your needs. <br>
+              If it is in your plan that you do not need such a large amount of food, then you can reduce the portion of what you buy. Always make a conscious effort to serve food in smaller portions. <br>
+              Even in restaurants and if you do happen to have more than what you can finish on the table, there is never a problem to take the food back as a take-away to finish it back home.</p>
+          </div>
+        </div>
+
```

Comment:
Above here, I simplified the content page to be more reader-friendly for our users and can select different tabs to find out more information of what they can do as an individual. With that, it reduces the "wordiness" of the content and we can direct the focus of our users to what information they want to find out more about.